### PR TITLE
THU/stub scenario runs

### PR DESCRIPTION
**Pull request by csm-thu**

### CODE REVIEW

**Review by esasova — APPROVED** [review]
some typos, LGTM :+1:

File: cypress/commons/constants/generic/TestConstants.js [hunk 1]
```diff
@@ -29,7 +29,9 @@ export const API_ENDPOINT = {
   SCENARIO: URL_ROOT + '/.*/scenarios/(s-[\\w]+)',
   SCENARIO_DEFAULT_SECURITY: URL_ROOT + '/.*/scenarios/(s-[\\w]+)/security/default',
   SCENARIO_SECURITY_ACL: URL_ROOT + '/.*/scenarios/(s-[\\w]+)/security/access',
-  RUN_SCENARIO: URL_ROOT + '/.*/scenarios/(s-[\\w]+)/run',
+  RUN_SCENARIO: URL_ROOT + '/.*/scenarios/(s-[\\w]+)/run', // Endpoint to start the run of a scenario
```
[the remark this answers is not in the document]
Comment (csm-thu) [reply cypress/commons/constants/generic/TestConstants.js:32]: Good idea, I'll do that :slightly_smiling_face:

File: cypress/commons/actions/generic/ScenarioParameters.js [hunk 2]
```diff
@@ -101,28 +101,34 @@ function cancelDiscardAndContinue() {
...
-  cy.intercept('PATCH', URL_REGEX.SCENARIO_PAGE_WITH_ID, (req) => {
-    req.on('response', (res) => {
-      res.setDelay(responseDelay);
-    });
-  }).as(alias);
-
+// Parameter 'options' is an object with the following properties:
+//  - scenarioId: scenario id to provide to the interception of the "get scenario" query (default: undefined)
+//  - runOptions: options to provide to the interception of the "get scenario run" query (default: undefined)
+//  - saveAndLaunch: boolean defining whether or not the test will trigger a SAVE of the scenario parameters (default:
```
[the remark this answers is not in the document]
Comment (csm-thu) [reply cypress/commons/actions/generic/ScenarioParameters.js]: Fixed, thank you for the feedback :slightly_smiling_face:

File: cypress/commons/actions/generic/ScenarioParameters.js [hunk 3]
```diff
@@ -1,10 +1,10 @@
 // Copyright (c) Cosmo Tech.
 // Licensed under the MIT license.
 
-import { URL_REGEX } from '../../constants/generic/TestConstants';
 import { GENERIC_SELECTORS } from '../../constants/generic/IdConstants';
 import { Scenarios } from '../generic';
 import { apiUtils as api } from '../../utils';
+import { stub } from '../../../commons/services/stubbing';
```
[the remark this answers is not in the document]
Comment (csm-thu) [reply cypress/commons/actions/generic/ScenarioParameters.js]: Done :+1:

**Pull request merged**